### DISCUSSION TITLE
I unable to edit the button problem for grapesjs-plugin-forms

### DISCUSSION THREAD

**Discussion by aharishsundhar** [GrapesJS/grapesjs#5157] [Q&A]
@artf  Here I am facing the issues please help me out
![Screenshot from 2023-06-01 17-37-13](https://github.com/GrapesJS/grapesjs/assets/72383148/0997960f-1137-4727-85b5-7ee8d9a6df8c)

**Comment by aharishsundhar** [comment]
previously using Send text its used type: textnode it easily edit directly into editor area but now traits side 
now its customize the values

**Reply by artf** [reply]
This solution prevents issues with editing button text directly via contentEditable (eg. space is not working). You can always replace/extend the button component with your own logic.
